source/install/self-managed-changelog.md

```diff
@@ -5,12 +5,154 @@
 See the [changelog in progress](https://bit.ly/2nK3cVf) for the upcoming release. See the [Legacy Self-Hosted Mattermost Changelog](legacy-self-hosted-changelog) for details on all Mattermost self-hosted releases prior to v6.0. 
 
 Latest Mattermost Releases:
+- [Release v7.2 - Feature Release](#release-v7-2-feature-release)
 - [Release v7.1 - Extended Support Release](#release-v7-1--extended-support-release)
 - [Release v7.0 - Major Release](#release-v7-0-major-release)
 - [Release v6.7 - Feature Release](#release-v6-7-feature-release)
-- [Release v6.6 - Feature Release](#release-v6-6-feature-release)
 - [Release v6.3 - Extended Support Release](#release-v6-3-extended-support-release)
 
+## Release v7.2 - [Feature Release](https://docs.mattermost.com/upgrade/release-definitions.html#feature-release)
+
+- **v7.2.0 release day: 2022-08-16**
+
+### Important Upgrade Notes
```

Comment:
Not sure if there is a way / need to format the below upgrade not better.

Reply:
> Also, has this been communicated to the customer support team?

Typically we just document the schema changes in upgrade notes. And in cases of major schema changes, we give the support team advance notice. IMO this is not major enough to separately ping the team.